ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -236,7 +240,13 @@ StackLayout {
 
         footer: Loader {
             sourceComponent: overviewSettingsFooterComp
-            active: !root.communitySettingsDisabled
+            active: {
+                if (root.communitySettingsDisabled)
+                    return false
+                if (root.isAdmin || root.isTokenMaster)
+                    return root.isPendingOwnershipRequest // not allowed for admin or TM unless there's the pending request
+                return true
+            }
```

Comment:
Just checking, is this logic correct @jrainville ? Pls approve if yes :)

Reply:
I think it's correct yes